ugoira/lib.py

```diff
@@ -58,7 +58,7 @@ def get_illust_url(illust_id: int) -> str:
 
     """
     return (
-        'http://www.pixiv.net/member_illust.php'
+        'https://www.pixiv.net/member_illust.php'
```

Comment:
please fix URL scheme in test codes

Reply:
I don't think that'd be necessary; I changed them to use get_illust_url in 7eb5c91b0, and the tests fail with the URLs fixed anyway. It would only introduce merge conflicts.